src/main/java/com/keeper/homepage/domain/seminar/application/SeminarAttendanceService.java

```diff
@@ -24,11 +27,15 @@
 @Transactional(readOnly = true)
 public class SeminarAttendanceService {
 
+  private final RedisUtil redisUtil;
   private final SeminarAttendanceRepository attendanceRepository;
   private final SeminarRepository seminarRepository;
 
   @Transactional
-  public SeminarAttendanceResponse save(Long seminarId, Member member, SeminarAttendanceCodeRequest request){
+  public SeminarAttendanceResponse save(Long seminarId, Member member, SeminarAttendanceCodeRequest request) {
+    String key = "seminar:" + LocalDate.now() + "memberId:" + member.getId();
```

Comment:
오오 redis key naming convention을 사용하셨군요..! 부끄럽게도 전 redis 만질 때 저걸 몰라서 key name을 막 지었습니다 ㅎㅎ...

nit)
```suggestion
    String key = "seminar:" + LocalDate.now() + ":memberId:" + member.getId();
```

Reply:
그러고 보니 야구게임, 로그인 쪽에서는 `_` 로 키 이름을 구분지어져 있네요! 통일성 있게 `_` 로 수정하겠습니다!

Reply:
아 아니에용 @shkisme 님의 naming  convention에 맞게 야구 게임쪽을 고치겠습니다~~